clang/test/Headers/stdckdint.c

```diff
@@ -1,12 +1,15 @@
 // NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 3
+// RUN: %clang_cc1 -triple=x86_64 -emit-llvm -verify -std=c99 %s -o - | FileCheck %s
```

Comment:
I don't add tests in clang/test/Headers/c11.c because I think here already covers and there are some specific flags/arguments in some of commands in c11.c.